packages/api/src/util/logger/logger.test.ts

```diff
@@ -0,0 +1,37 @@
+// @ts-ignore
+global.console = {
+    log: jest.fn(),
+    info: jest.fn(),
+    // error: jest.fn()
+    error: global.console.error
+};
+
+const someDate = new Date("2019-10-01 23:18:02");
+// @ts-ignore
+global.Date = class extends Date {
+    constructor() {
+        // @ts-ignore
+        return super(someDate);
+    }
+}
```

Comment:
Some mocks are a bit tricky to setup in jest. This could be shorter if we used dependencies like [jest-date-mock](https://www.npmjs.com/package/jest-date-mock) and [std-mocks](https://github.com/neoziro/std-mocks) but I'm not sure it's worth it.